packages/reusable-blocks/src/components/reusable-blocks-menu-items/reusable-block-convert-button.js

```diff
@@ -39,11 +40,13 @@ export default function ReusableBlockConvertButton( {
 	rootClientId,
 } ) {
 	const [ isModalOpen, setIsModalOpen ] = useState( false );
+	const [ blockType, setBlockType ] = useState( 'resuable' );
 	const [ title, setTitle ] = useState( '' );
-	const canConvert = useSelect(
+	const [ categoryName, setCategoryName ] = useState( '' );
```

Comment:
Setting the initial state for `categoryName` to `''` without providing a matching placeholder to the category select options makes it look like `Banners` is selected so if the user doesn't change the selection, the pattern is created but with no category selected.

It might also be worth noting that the pattern categories are registered async so the results of the `useSelect` below will start out as an empty array making using the first category as a default more involved.

Reply:
I have pushed a fix that partly solves this which is to default to a `Select a category` option in the select list, but there are a few questions we need to answer in order to get a full fix for this:

1. Do we want to make the selection of a category required, eg. disable the `Save` button until one is selected. Is a category required for theme and .org patterns?
2. If it is required should we add an `Uncategorized` option for when people can't see a suitable catgory
3. Also if it is required we probably need to allow users to add new categories somehow - I am currently looking at WP taxonomies to see if that is a better way to handle these categories